app/workers/backend_storage_rewrite_worker.rb

```diff
@@ -4,21 +4,6 @@ class BackendStorageRewriteWorker
   include Sidekiq::Worker
   sidekiq_options queue: :low
 
-  ENQUEUER = ->(class_name, ids) { BackendStorageRewriteWorker.perform_async(class_name, ids) }
-
-  # Enqueue async processing for all providers
-  def self.enqueue_all
-    Backend::StorageRewrite::AsyncProcessor.new(enqueuer: ENQUEUER).rewrite_all
-  end
-
-  # Enqueue async processing for a single provider
-  def self.enqueue(provider_id)
-    Backend::StorageRewrite::AsyncProcessor.new(enqueuer: ENQUEUER).rewrite_provider(provider_id)
-  end
-
-  # The arguments of the worker are: the class name of the collection, and the array of IDs of objects of this class
-  # It should not exceed the BATCH_SIZE defined in AsyncProcessor
-  # All objects in a batch will belong to the same provider (tenant)
```

Comment:
@akostadinov did you remove the comment on purpose?

Some clarification was requested by @jlledom

Reply:
I don't see how worker cares whether IDs are from same provider or not? Will it fail? Do we want to avoid that and why so? I see no value in this comment. It creates more questions for me than answers.

What this comment does at best is expain a little what `AsyncProcessor` does. But then why do that. One can look at `AsyncProcessor` and see how enqueuing is being done. Very easy to grep that too.